### PR TITLE
fix(taskframework): refresh entity from database after update destroy time

### PR DESCRIPTION
**Pull request by krihy**
#### Which issue(s) this PR fixes:

fix get entity from database, avoid get result form jpa cache 

#### Additional documentation e.g., usage docs, etc.:

```docs

```

### CODE REVIEW

File: server/odc-migrate/src/main/resources/migrate/common/V_4_2_4_12__update_system_config.sql [hunk 1]
```diff
@@ -1,4 +1,6 @@
 update `config_system_configuration` set `value` = 'true' where `key` = 'odc.task-framework.enabled';
 update `config_system_configuration` set `value` = '3' where `key` = 'odc.task-framework.executor-waiting-to-run-threshold-seconds';
 update config_system_configuration set `value`='0/10 * * * * ?' where `key` = 'odc.task-framework.check-running-job-cron-expression';
+update config_system_configuration set `value`='0/1 * * * * ?' where `key` = 'odc.task-framework.do-canceling-job-cron-expression';
```
Comment (guowl3) [inline server/odc-migrate/src/main/resources/migrate/common/V_4_2_4_12__update_system_config.sql:4]: Can you provide a soft exit interface without directly killing the process?

File: server/odc-service/src/main/java/com/oceanbase/odc/service/task/service/StdTaskFrameworkService.java [hunk 2]
```diff
@@ -114,6 +115,13 @@ public JobEntity find(Long id) {
                 .orElseThrow(() -> new NotFoundException(ResourceType.ODC_TASK, "id", id));
     }
 
+    @Override
+    public JobEntity findFromDatabase(Long id) {
```
Comment (guowl3) [inline server/odc-service/src/main/java/com/oceanbase/odc/service/task/service/StdTaskFrameworkService.java:119]: This is a duplicate method; we could merge it with the other one.

**Review by guowl3 — APPROVED** [review]
LGTM

**Review by yizhouxw — APPROVED** [review]

**Review by yhilmare — DISMISSED** [review]
lgtm

**Review by yhilmare — APPROVED** [review]
lgtm

**Pull request merged**